internal/sponge/sfix/sfix.go

```diff
@@ -0,0 +1,50 @@
+package sfix
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/coralproject/shelf/internal/sponge"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	mgo "gopkg.in/mgo.v2"
```

Comment:
We need to figure this out because this is unnecessary and will create inconsistencies. What program are you using for gofmt?  goimports, goreturn?